specs/waku/v2/waku-relay-security-analysis.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Security Analysis of Waku Relay
+version: 
+status: Draft
+authors: Sanaz Taheri <sanaz@status.im>
+---
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+  - [Terminologies](#terminologies)
+  - [Anonymity](#anonymity)
+    - [Definition](#definition)
+      - [Message Publisher Anonymity](#message-publisher-anonymity)
+      - [Topic Subscriber Anonymity](#topic-subscriber-anonymity)
+    - [Analysis](#analysis)
+  - [Authenticity](#authenticity)
+    - [Definition](#definition-1)
+    - [Analysis](#analysis-1)
+  - [Integrity](#integrity)
+    - [Definition](#definition-2)
+    - [Analysis](#analysis-2)
+  - [Confidentiality](#confidentiality)
+    - [Definition](#definition-3)
+    - [Analysis](#analysis-3)
+  - [Changelog](#changelog)
+    - [2.0.0-beta1](#200-beta1)
+- [Copyright](#copyright)
+- [References](#references)
+
+# Abstract
+
+In this spec, we analyze the security of the  `relay` protocol concerning data confidentiality, integrity, authenticity, and anonymity. This is to enable users of this protocol to make informed decision about all the secuirty properties that can or can not achieve by deploying `relay` protocol. 
```

Comment:
I think this should be part of the existing relay spec, we want to be clear about guarantees in that spec, not just analyze it from a far with a different spec (which also can't really be implemented, more of an analysis/guarantees)

Reply:
> I think this should be part of the existing relay spec, we want to be clear about guarantees in that spec, not just analyze it from a far with a different spec (which also can't really be implemented, more of an analysis/guarantees)

Sure, addressed!